components/constellation/BUILD.gn

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
please add README.md explaining what this component is/does

Reply:
also does this need to be a top-level component or can it go inside p3a? Are we planning to use this somewhere outside of p3a?

Reply:
There are no plans to use this outside of P3A at the moment. I was copying the scheme used by other Rust/Cxx/FFI bindings like adblock, etc. Since both you and @rillian raised the same concern, I will move `constellation` and `nitro_utils` into the `p3a` component.